Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -135,7 +135,8 @@ variable [DecidableEq n] {A : Matrix n n 𝕜} (hA : PosSemidef A)
 /-- The positive semidefinite square root of a positive semidefinite matrix -/
 @[pp_dot]
 noncomputable def sqrt : Matrix n n 𝕜 :=
-  hA.1.eigenvectorMatrix * diagonal ((↑) ∘ Real.sqrt ∘ hA.1.eigenvalues) * hA.1.eigenvectorMatrixᴴ
+  hA.1.eigenvectorUnitary.1 * diagonal ((↑) ∘ Real.sqrt ∘ hA.1.eigenvalues) *
+  hA.1.eigenvectorUnitaryᴴ
```

Comment:
I think for elements of `unitaryGroup` we should prefer this spelling over `Matrix.conjTranspose`.
```suggestion
  (star hA.1.eigenvectorUnitary : Matrix n n 𝕜)
```